setuptools/py27compat.py

```diff
@@ -4,7 +4,7 @@
 
 import platform
 
-import six
+from setuptools.extern import six
```

Comment:
`six` is already used elsewhere, so it makes sense to use it here as well (after the import is corrected).

@westsouthnight This was worked around by pulling the affected version from PyPI, so things should be fine again for now.